index.html

```diff
@@ -444,7 +450,7 @@ <h2><dfn>observe()</dfn> method</h2>
             <li>If <var>options</var>'s <a>buffered</a> flag is set:
               <ol>
                 <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
-                object returned by <a href="#filter-buffer-by-name-and-type"></a>
+                object returned by <a href="#filter-buffers-by-name-and-type"></a>
                 algorithm with <var>buffer</var> set to <a>relevant
```

Comment:
Again here we would like consistency with the change in the input to this algorithm.

Reply:
Again this is only a single type, so I think it should be singular.

Reply:
Nicolas and I chatted offline about making sure the variables line up this should be resolved.